client/src/app/pages/assessment-management/assessment-settings/assessment-settings-page.tsx

```diff
@@ -283,19 +318,21 @@ const AssessmentSettings: React.FC = () => {
                             width={10}
                             {...getTdProps({ columnKey: "questions" })}
                           >
-                            {questionnaire.questions}
+                            {calculateTotalQuestions(questionnaire)}
```

Comment:
Having a helper component that also does the calc would be a reasonable alternative.  That's how the archetype table is structured.  Eg:

```jsx
const QuestionnaireQuestionsColumn: React.FC<{ questionnaire: Questionnaire }> = ({
  questionnaire,
}) => {
  const totalQuestions = (questionnaire.sections || []).reduce((total, section) => {
    return total + (section.questions ? section.questions.length : 0);
  }, 0);
  return <Text>{totalQuestions}</Text>;
};
```

The `totalQuestions` calc could be wrapped in a `useMemo()`, but that would probably be overkill.